src/components/ProductsList.js

```diff
@@ -1,14 +1,21 @@
 import React from "react";
 import { Link } from "react-router-dom";
 
-export default function ProductsList({ products }) {
+export default function ProductsList({ products, listType }) {
   return (
     <>
-      <h2>Products</h2>
+      {listType === "bestRated" ? (
```

Comment:
component is better reusable if the heading is not included. e.g. h2 might be the wrong level of heading.
this way the title can be independent of the lists loading state